backend/tracim_backend/views/core_api/workspace_controller.py

```diff
@@ -167,6 +167,7 @@ def update_workspace(self, context, request: TracimRequest, hapic_data=None):  #
             agenda_enabled=hapic_data.body.agenda_enabled,
             save_now=True,
         )
+        wapi.execute_update_workspace_actions(request.current_workspace)
```

Comment:
remove this

Reply:
see : https://github.com/tracim/tracim/pull/1578#discussion_r274287675

Reply:
when reading the code, I must be warned that this way of using the API is broken by design.

Reply:
added a warning in code.